tensorflow/core/lib/bfloat16/bfloat16.h

```diff
@@ -21,7 +21,7 @@ limitations under the License.
 
 #include "tensorflow/core/platform/byte_order.h"
 
-#ifdef __CUDACC__
+#if defined(__CUDACC__) || __HIP__
```

Comment:
Is this missing a `defined()` around the `__HIP__`?  

e.g.
```
#if defined(__CUDACC__) || defined(__HIP__)
```

Reply:
it doesn't matter according to C/C++ spec:
https://stackoverflow.com/questions/5085392/what-is-the-value-of-an-undefined-constant-used-in-if